[SSL & SSL4SaaS] Delegated DCV (#8168)
cloudflare/cloudflare-docs: content/ssl/_partials/_partial-zone-acm-dcv-nonwildcard.md

  list: never
---

If every hostname on a non-wildcard certificate is [proxying traffic](/dns/manage-dns-records/reference/proxied-dns-records/) through Cloudflare, Cloudflare can automatically complete DCV on your behalf.

This applies to customers using [Universal](/ssl/edge-certificates/universal-ssl/) or [Advanced certificates](/ssl/edge-certificates/advanced-certificate-manager/).

> However, at least one hostnames on the certificate is not proxying traffic through Cloudflare, certificate issuance and renewal varies based on the type of certificate you are using:

Reviewer: I'm not entirely sure what the meaning here is. Is it `If one of the hostnames on the certificate is not proxying traffic through Cloudflare, then certificate issuance and renewal will vary based on the type of certificate you are using:`

Is this it?

> However, at least one hostnames on the certificate is not proxying traffic through Cloudflare, certificate issuance and renewal varies based on the type of certificate you are using:

Author: ```suggestion
If one of the hostnames on the certificate is not proxying traffic through Cloudflare, certificate issuance and renewal will vary based on the type of certificate you are using:
```
